app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
+    has_many :comments
```

Comment:
You could add `dependent: :destroy` so that if a post is removed the comments for that post are removed as well.